src/wisdomClient.ts

```diff
@@ -23,7 +24,8 @@ export type ServiceInputTypes =
   | ListIntegrationAssociationsInput
   | NotifyRecommendationsReceivedInput
   | QueryAssistantInput
-  | SearchSessionsInput;
+  | SearchSessionsInput
+  | GetContactInput;
```

Comment:
could consider placing the ';' (semicolon) on a separate line to avoid having the line removal